tests/vmi_kernel_boot_test.go

```diff
@@ -122,4 +124,65 @@ var _ = Describe("[sig-compute]VMI with external kernel boot", func() {
 			Expect(err.Error()).To(ContainSubstring("denied the request: spec.domain.firmware.kernelBoot.container must be defined with at least one of the following: kernelPath, initrdPath"))
 		})
 	})
+
+	Context("with external alpine-based kernel only (without initrd)", func() {
+		getVMIKernelBoot := func() *kubevirtv1.VirtualMachineInstance {
+			vmi := utils.GetVMIKernelBoot()
+			// Remove initrd path from vmi spec
+			kernelBoot := vmi.Spec.Domain.Firmware.KernelBoot
+			kernelBoot.Container.InitrdPath = ""
+			return vmi
+		}
+
+		It("ensure successful boot", func() {
+			vmi := getVMIKernelBoot()
+
+			obj, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+			Expect(err).ToNot(HaveOccurred())
+			tests.WaitForSuccessfulVMIStart(obj)
```

Comment:
Don't you need to login here to verify if it is actually booted? Same below.

Reply:
Is the VM now actually booting into the OS or would it need the initrd too? If we would need a specific kernel to test it, I guess it would be ok to go without, since that would be quite some overhead for @daghaian. If it boots into the OS, I agree.

Reply:
Yeah I think we would need a new kernel.  Without this patch we don't get to running state.  To me that is good enough.

Reply:
Also @xpivarc the tests with initrd don't login either

Reply:
@rmohr We can check to identify whether or not we're actually making it into the OS but as @mhenriks mentioned already, it would not appear that any login checks are being performed even with the original kernel+initrd scenario. If we want to modify both the test involving this proposed patch and the previous, we can discuss that further.

Reply:
Fine by me. Let's see what @xpivarc says.

Reply:
I apologize I was interrupted in the morning when I was trying to reply.

I was under the impression we have it(but maybe we had a custom kernel for something else...).
I have only one objection and that is the name of the test `ensure successful **boot**` which might be confusing in the future.

Thank you for the fix!

Reply:
@xpivarc what, exactly is confusing?  The other test, which is exactly the same, but with initrd, has the same name.  I think we should be extra friendly to new contributors and we're perhaps being a little too pedantic here.

Reply:
Forgot to unhold. 
@mhenriks Well, the other test is confusing to me too. We can clean it up altogether(in the next one). The confusing part is the name. It should tell me what the test is trying to test. I would expect it to verify that OS is correctly booted if it says so. (This test however only verifies we do not crash) It happened to me at least once that the test was actually not testing what it was stating and it can confuse a lot...

I was holding only because I thought we have a possibility to test this better. Thank you for correcting me.